docs/_guide/attrs.md

```diff
@@ -0,0 +1,204 @@
+
+```js
+import { controller, attr } from "@github/catalyst"
+
+@controller
+class HelloWorldElement extends HTMLElement {
+  @attr foo = 1
+
+  connectedCallback() {
+    console.assert(this.getAttribute('data-foo') === '1')
+    this.setAttribute('data-foo', 'not a number')
+    console.assert(Number.isNaN(this.foo))
+    this.foo = -3.14
+    console.assert(this.getAttribute('data-foo') === '-3.14')
+  }
+}
+```
+
+### Default Values
+
+When an element gets connected to the DOM, the attr is initialized. During this phase Catalyst will determine if the default value should be applied. The default value is defined in the class property. The basic rules are as such:
+
+ - If the class property has a value, that is the _default_
+ - When connected, if the element _does not_ have a matching attribute, the default _is_ applied.
+ - When connected, if the element _does_ have a matching attribute, the default _is not_ applied, the property will be assigned to the value of the attribute instead.
+
+<div class="d-flex border rounded-1 my-3 box-shadow-medium">
+  <span class="d-flex flex-items-center bg-blue text-white rounded-left-1 p-3">
+    <svg width="24" height="24" viewBox="0 0 14 16" class="octicon octicon-info" aria-hidden="true">
+      <path
+        fill-rule="evenodd"
+        d="M6.3 5.69a.942.942 0 0 1-.28-.7c0-.28.09-.52.28-.7.19-.18.42-.28.7-.28.28 0 .52.09.7.28.18.19.28.42.28.7 0 .28-.09.52-.28.7a1 1 0 0 1-.7.3c-.28 0-.52-.11-.7-.3zM8 7.99c-.02-.25-.11-.48-.31-.69-.2-.19-.42-.3-.69-.31H6c-.27.02-.48.13-.69.31-.2.2-.3.44-.31.69h1v3c.02.27.11.5.31.69.2.2.42.31.69.31h1c.27 0 .48-.11.69-.31.2-.19.3-.42.31-.69H8V7.98v.01zM7 2.3c-3.14 0-5.7 2.54-5.7 5.68 0 3.14 2.56 5.7 5.7 5.7s5.7-2.55 5.7-5.7c0-3.15-2.56-5.69-5.7-5.69v.01zM7 .98c3.86 0 7 3.14 7 7s-3.14 7-7 7-7-3.12-7-7 3.14-7 7-7z"
+      />
+    </svg>
+  </span>
+  <div class="p-3">
+
+Remember! The values defined in the class field are the _default_. They won't be set if the element is created and its attribute set to a custom value!
+
+  </div>
+</div>
+
+The following example illustrates this behavior:
+
+```js
+import { controller, attr } from "@github/catalyst"
+@controller
+class HelloWorldElement extends HTMLElement {
+  @attr name = 'World'
+  connectedCallback() {
+    this.textContent = `Hello ${name}`
+  }
+}
+```
+
+```html
+<hello-world></hello-world>
+// This will render `Hello World`
+
+<hello-world data-name="Catalyst"></hello-world>
+// This will render `Hello Catalyst`
+
+<hello-world data-name=""></hello-world>
+// This will render `Hello `
+```
+
+### What about without Decorators?
+
+If you're using decorators, then you won't be able to use the `@attr` decorator, but there is still a way to achieve the same result. Under the hood `@attr` simply tags a field, but `initializeAttrs` does all of the logic.
```

Comment:
Should this sentence start with "If you're **not** using decorators, then ..."
Perhaps I'm not getting the meaning

Reply:
Another great catch :smile: 

```suggestion
If you're not using decorators, then you won't be able to use the `@attr` decorator, but there is still a way to achieve the same result. Under the hood `@attr` simply tags a field, but `initializeAttrs` does all of the logic.
```